easybuild/easyconfigs/g/GRASS/GRASS-7.0.2-foss-2015a.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'ConfigureMake'
+
+name = "GRASS"
+version = "7.0.2"
+
+homepage = "http://grass.osgeo.org"
+description = """The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization"""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'openmp': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://grass.osgeo.org/grass%s/source' % ''.join(version.split('.')[:2])]
+
+patches = [
+    '%(name)s-%(version)s_OpenMP.patch',
+    '%(name)s-%(version)s_ncurses.patch',
+]
+
+dependencies = [
+    ('Python', '2.7.11'),        # and so png, curses, readline
+    ('libpng', '1.6.20'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('netCDF', '4.3.3.1'),
+    ('GDAL', '2.0.1'),
+    ('libspatialite', '4.3.0a'), # and so GEOS, SQLite
+    ('freetype', '2.6.2'),
+    ('FFmpeg', '2.8.5'),
+    ('LibTIFF', '4.0.6'),
+]
+
+preconfigopts = "LDFLAGS=' -lm -lpthread ' "
+
+configopts = """\
+    --with-cxx  --enable-64bit  --enable-largefile=yes \
+    --without-postgres  --with-sqlite  --with-spatialite \
+    --with-blas  --with-lapack  --with-fftw  --with-pthread \
+    --with-openmp \
+    --with-geos \
+    --with-readline \
+    --with-freetype \
+    --with-netcdf \
+    --with-ffmpeg \
+    --with-ffmpeg-includes=$EBROOTFFMPEG/include/* \
+    --with-proj-share=$EBROOTPROJ/share/proj \
+    --without-opengl \
+    --without-glw \
+    --without-tcltk \
+    --with-python \
+    --without-wxwidgets \
+    --without-x \
+    --with-png \
+    --with-png-libs=$EBROOTLIBPNG/lib
+"""
+
+prebuildopts = "sed -i -e '/^\s*gui\s*\\\s*$/d' Makefile && rm -rf ./gui && "
```

Comment:
@valtandor the tests are failing because the `prebuildopts` value isn't the same after the parsed easyconfig file has been dumped again... this may be a bug in the dumping, but it does indicate that nasty things are going on

What are you trying to do here exactly?